src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@ public interface MovieDao {
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
```

Comment:
Consider using the primitive `boolean` instead of `Boolean` for the `update` method's return type to avoid unnecessary boxing and potential nullability issues.